### PR TITLE
Implemented password feature

### PR DESCRIPTION
**Pull request by PradipDeore**

### CODE REVIEW

File: src/main/java/com/example/accessingdatamysql/DemoBss.java [hunk 1]
```diff
@@ -16,4 +16,9 @@ public Response login3(@RequestParam() String username, @RequestParam String pas
 return null;
 
     }
```
Comment (PradipDeore) [inline src/main/java/com/example/accessingdatamysql/DemoBss.java:18]: Please add code documentation

**Comment by bhushansinkar** [comment]
not satisfied, correct it

**Comment by PradipDeore** [comment]
Fixed done as needed

**Comment by bhushansinkar** [comment]
done

**Pull request merged**